File: ChatDemo-UI3.0/ChatDemo-UI3.0/Code/Model/EMMessageModel.m
/************************************************************
 *  * Hyphenate
 * __________________
 * Copyright (C) 2016 Hyphenate Inc. All rights reserved.
 *
 * NOTICE: All information contained herein is, and remains
 * the property of Hyphenate Inc.
 */

#import "EMMessageModel.h"

@implementation EMMessageModel

- (instancetype)initWithMessage:(EMMessage*)message
{
    self = [super init];
    if (self) {
        _message = message;
    }
    return self;
}

@end

File: swift-demo/Hyphenate-Demo-Swift/Code/Models/EMMessageModel.swift
//
//  EMMessageModel.swift
//  Hyphenate-Demo-Swift
//
//  Created by dujiepeng on 2017/6/19.
//  Copyright © 2017 dujiepeng. All rights reserved.
//

import UIKit
import Hyphenate

class EMMessageModel: NSObject {
    var message: EMMessage?
    var isPlaying: Bool = false
    var thumbnailImage: UIImage?
    
    init(withMesage msg: EMMessage) {
        message = msg
    }
}
